apps/namadillo/src/types/txKind.ts

```diff
@@ -0,0 +1,13 @@
+export const txKinds = [
```

Comment:
I think we need to discuss how we're going to organize the types in the codebase. One advantage of having them in a root folder is that is easy to find, import, and maintain. Do you think we should break the types in different files?

Reply:
I actually did this just because I needed a `const` and I couldn't do that in `types.d.ts`. Now that your PR changes it to `types.ts`, we could still use that but I'll leave that until we discuss how to handle it.